### ISSUE TITLE
area_extent unit scale

### ISSUE THREAD

**Issue by anikfal** [pytroll/pycoast#31]
#### Code:
```python
from PIL import Image
from pycoast import ContourWriterAGG
img = Image.open('BMNG_clouds_201109181715_areaT2.png')
proj4_string = '+proj=stere +lon_0=8.00 +lat_0=50.00 +lat_ts=50.00 +ellps=WGS84'
area_extent = (-3363403.31,-2291879.85,2630596.69,2203620.1)
area_def = (proj4_string, area_extent)
...

```
#### Question:

What is the unit scale of the values of `area_extent`?

**Comment by djhoese** [comment]
The area extents are the coordinates for the outer edges of the outer pixels in the projected space. Typically these are meters. You can probably find more useful information on the pyresample documentation: https://pyresample.readthedocs.io/en/latest/geometry_utils.html

Let me know if this doesn't answer your question. We can use this issue as a sign that we need to update the documentation.

**Comment by anikfal** [comment]
Thank you for your reply. Seems to be the exact documentation I needed.
However, I think it is necessary to add this information in the `pycoast`'s documentation and user manual. I was frustrated in my attempts to find any bit of information about the unit scale.

**Comment by djhoese** [comment]
> However, I think it is necessary to add this information in the pycoast's documentation and user manual.

Agreed. It is a concept carried over from the pyresample package (and projections in general) and since people have often used both together (or through satpy) there was less work put in to pycoast's documentation. If you have any ideas of how this can be updated, feel free to make a pull request for the changes you think could help users in the future. Otherwise, this will have to wait until someone has the time.

**Comment by anikfal** [comment]
> feel free to make a pull request for the changes you think could help users in the future.

Sure. Thank you.
